integration-tests/fake_spine/fake_spine/vnp_responses/async_reliable_inbound.mustache

```diff
@@ -15,7 +15,7 @@ Content-Type: text/xml
+<soap:Envelope xmlns:xsi="http://www.w3c.org/2001/XML-Schema-Instance" xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:eb="http://www.oasis-open.org/committees/ebxml-msg/schema/msg-header-2_0.xsd" xmlns:xlink="http://www.w3.org/1999/xlink"><soap:Header><eb:MessageHeader eb:version="2.0" soap:mustUnderstand="1"><eb:From><eb:PartyId eb:type="urn:nhs:names:partyType:ocs+serviceInstance">YES-0000806</eb:PartyId></eb:From><eb:To><eb:PartyId eb:type="urn:nhs:names:partyType:ocs+serviceInstance">{{to_party_id}}</eb:PartyId></eb:To><eb:CPAId>S918999404212</eb:CPAId><eb:ConversationId>{{conversation_id}}</eb:ConversationId><eb:Service>urn:nhs:names:services:psis</eb:Service><eb:Action>MCCI_IN010000UK13</eb:Action><eb:MessageData><eb:MessageId>{{message_id}}</eb:MessageId><eb:Timestamp>2020-03-19T14:05:22Z</eb:Timestamp><eb:RefToMessageId>{{ref_to_message_id}}</eb:RefToMessageId></eb:MessageData><eb:DuplicateElimination/></eb:MessageHeader><eb:AckRequested eb:version="2.0" soap:mustUnderstand="1" soap:actor="urn:oasis:names:tc:ebxml-msg:actor:toPartyMSH" eb:signed="false"/><eb:SyncReply eb:version="2.0" soap:mustUnderstand="1" soap:actor="http://schemas.xmlsoap.org/soap/actor/next"/></soap:Header><soap:Body><eb:Manifest xmlns:hl7ebxml="urn:hl7-org:transport/ebXML/DSTUv1.0" eb:version="2.0"><eb:Reference xlink:href="cid:{{message_id}}@spine.nhs.uk"><eb:Schema eb:location="urn:hl7-org:v3_MCCI_IN010000UK13.xsd" eb:version="13"/><eb:Description xml:lang="en">The HL7 payload</eb:Description><hl7ebxml:Payload style="HL7" encoding="XML" version="3.0"/></eb:Reference></eb:Manifest></soap:Body></soap:Envelope>
```

Comment:
Should we use https for schemas.xmlsoap.org

Reply:
These were scraped from actual opentest replies. I think we should use the same as opentest.